_posts/2021-06-01-migration-v1-v2.md

```diff
@@ -0,0 +1,130 @@
+
+### New REST API
+We changed the REST API layout to ensure better long term maintainability and because we refactored our API to have support for artifact groupings. Apicurio Registry still supports the original REST API (now known as `v1`) and the various compatibility APIs we already supported (confluent compat, ibm compat).  Additionally, Registry now also implements the Schema Registry spec provided within the CNCF Cloud Events spec.
+
+### Refactored SerDes libraries
+The Serializer and Deserializer classes are now available as three different maven modules (one for each data format).  This makes it possible to use just the one you want without pulling in a bunch of transitive dependencies that you don't care about.
+
+## Migration
+
+The migration from Apicurio Registry 1.3.x to 2.x requires you to move the data from your old registry to another along with a review of your applications that interact with the registry in order to update the configuration of your applications to accomadate to the new requirements.
+
+### Data Migration
+
+The process of data migration to Apicurio Registry 2.x will require you to export all the data from your 1.3.x deployment and import it to the new 2.x based deployment.
+
+Data migration is a critical step in the migration to Apicurio Registry 2.x if you are using the registry as a schema registry for Kafka applications, because every Kafka message carries the global identifier for the schema stored in Apicurio Registry. It is critical that this identifier is kept between registry upgrades via data migration.
+
+For that purpose Apicurio Registry 2.x provides a new import/export API to bulk import or export all the data from your registry deployment. This API guarantees that all the identifiers are kept when importing the data from your old registry. The export API works by downloading a custom zip file containing all the information of your artifacts. The import API accepts that zip file, loading all of the artifacts into the registry in a single batch.
+
+Apicurio Registry 1.3.x does not provide an import/export API but with the 2.x release we provided an export tool compatible with Apicurio Registry 1.3.x that allows you to export a compatible zip file which then can be imported to your 2.x registry.  This special export tool uses common existing APIs to export all of the content in the registry.  It is less performant than the built-in export API we have introduced in 2.x, and so should only be used when exporting from a 1.3.x registry.
+
+This export tool can be found [here](https://github.com/Apicurio/apicurio-registry/tree/2.0.x/utils/exportV1). It is a java application meant to be invoked from the command line.
```

Comment:
Suggest "This export tool is available from the [Apicurio Registry GitHub project] ...". This is a Java application which is..."